### DISCUSSION TITLE
MAUI Android Samsung

### DISCUSSION THREAD

**Discussion by jbloink** [thudugala/Plugin.LocalNotification#356] [Q&A] [answered]
I've tested notifications in the Android emulator (on Visual Studio 2022 17.5.0 Preview 2.0) and everything seems to work great. I have tested it recurring every 5 minutes and it works as expected. I've tried closing the app on the emulator and the notification still fires (as expected).

When I connect my Samsung Galaxy S22+ and deploy to it through Visual Studio 2022, the notifications never fire. Is this a known issue with Samsung phones or do I need to do something different when deploying to a physical device? When I install the app on the physical device, it prompts if I want to enable notifications, which I accept. I am deploying in debug mode, which might also be the issue. 

Any guidance is greatly appreciated. I've been trying to research/troubleshoot this for the past couple of days but I can't find much on it.

I have this in my MauiProgram.cs, and my physical device is Android 13 (API v33).

#if IOS || ANDROID33_0_OR_GREATER
            .UseLocalNotification()
#endif

Below is my code. I am targeting .NET7. 

public async void ConfigureNotifications()
        {
            if (await LocalNotificationCenter.Current.AreNotificationsEnabled() == false)
            {
                await LocalNotificationCenter.Current.RequestNotificationPermission();
            }

            //Set the notification for 12pm on Sundays
            var scheduleddatetime = DateTime.Now;
            var nextSunday = scheduleddatetime.AddDays(7 - (int)scheduleddatetime.DayOfWeek).Date;
            var nextSundayAtNoon = nextSunday.AddHours(12);

            var notification = new NotificationRequest
            {
                NotificationId = 100,
                Title = "MKIPA",
                Description = "Be sure to log your IPAs for the week",
                ReturningData = "Dummy data", 
                Schedule =
                {
                    NotifyTime = nextSundayAtNoon, 
                    NotifyRepeatInterval = TimeSpan.FromDays(7)
                }
            };
            await LocalNotificationCenter.Current.Show(notification);
        }

Thanks in advance!
Jason

**Comment by thudugala** [comment, marked as answer]
@jbloink Have you confirmed notification setting is actually enabled?

**Reply by jbloink** [reply]
@thudugala - you are correct, notifications weren't enabled, even though I enabled them when the app first launched. I assumed by enabling them during the initial install they would be enabled, but they were not for some reason. Thanks for the follow up!

**Reply by adriennobel** [reply]
Please how did you enable the notifications on Samsung Galaxy s22 because I have the same exact concern as you.
